.gitignore

```diff
@@ -43,3 +43,9 @@ dkeycache/App/auto_version.h
 dkeyserver/App/auto_version.h
 
 test/__pycache__/
```

Comment:
commit msg:
"
Add pair-wise test for asymmetric key pair 

for the asymmetric key generation, only it passed the pair-wise test,
the key pair can be released to the caller.

test: passed all the unit test
Signed-off-by: wanghaiheng <haihengx.wang@intel.com>
"

Reply:
Signed-off-by: wanghaiheng <haihengx.wang@intel.com>

Reply:
ok